src/renderer/link-tooltip.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2024-2025 New Vector Ltd.
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+SPDX-License-Identifier: AGPL-3.0-only OR GPL-3.0-only OR LicenseRef-Element-Commercial
+Please see LICENSE files in the repository root for full details.
+*/
+
+import React from "react";
+import { domToReact } from "html-react-parser";
+
+import LinkWithTooltip from "../components/views/elements/LinkWithTooltip";
+import { getSingleTextContentNode, type RendererMap } from "./utils.tsx";
+
+export const ambiguousLinkTooltipRenderer: RendererMap = {
```

Comment:
These should probably all have doc, although some are fairly obvious. This one maybe a bit less obvious.